docs/api/navigators/StackNavigator.md

```diff
@@ -118,7 +118,7 @@ All `navigationOptions` for the `StackNavigator`:
 
 - `title` - a title (string) of the scene
 - `header` - a config object for the header bar:
-  - `visible` - Boolean toggle of header visibility. Only works when `headerMode` is `screen`.
+  - `bar` - React Element to display as a header or `null` to hide it. Only works when `headerMode` is `screen`
```

Comment:
What happens when there's nothing to go back to? Do you render a non-functional button? Also how'd you render something like previous screen title in back button>